README.md

```diff
@@ -4,8 +4,7 @@
 *** PyAEDT README
 -->
 
-
-# PyAEDT
+[![PyAEDT logo](https://github.com/ansys/pyaedt/blob/main/doc/source/_static/logo.png)](https://aedt.docs.pyansys.com)

```

Comment:
Always use absolute links in here. This is important. It avoids breaking the README.md in PyPI descriptions.